### PR TITLE
Chronos: Turn off the Forecaster auto-save checkpoint method

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def __init__(self, num_processes: int = 1,
                  use_ipex: bool = False,
                  enable_bf16=False,
                  distributed_backend="spawn",
+                 find_unused_parameters = False,
```
Comment (TheaperDeng) [inline python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:43]: drop this change

File: python/chronos/src/bigdl/chronos/forecaster/__init__.py [hunk 2]
```diff
@@ -25,23 +25,23 @@
     import torch
     torch_available = True
 except:
-    warnings.warn("Please install `torch` to use full collection of forecasters.")
+    warnings.warn(r"Please install `torch` to use forecasters, including tcmf\tcn\seq2seq\lstm.")
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/forecaster/__init__.py:28]: including TCMFForecaster, TCNForecaster, Seq2SeqForecaster, LSTMForecaster

File: python/chronos/src/bigdl/chronos/forecaster/__init__.py [hunk 3]
```diff
@@ -25,23 +25,24 @@
...
-    warnings.warn("Please install `torch` to use full collection of forecasters.")
+    warnings.warn("Please install `torch` to use forecasters, including TCMFForecaster, "
+                  "TCNForecaster, LSTMForecaster, Seq2SeqForecaster.")
 try:
     import tensorflow as tf
     assert tf.__version__ < "2.0.0"
     tf_available = True
 except:
-    warnings.warn("Please install `tensorflow<2.0.0` to use full collection of forecasters.")
+    warnings.warn("Please install `tensorflow<2.0.0` to use MTNETForecaster.")
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/forecaster/__init__.py:35]: MTNetForecaster

**Review by TheaperDeng — APPROVED** [review]
Others LGTM

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/190/

**Pull request merged**